index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my first portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100;200;300;500&family=Roboto:ital@1&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <section class="header">
+         <h2><a href="#" class="logo">softkoda</a></h2>   
+        <nav class="nav-bar">
+            <ul>
+                <li><a href="#"><img src="images/Menu.png" alt=""></a></li>
+            </ul>
+        </nav>
+    </section>
+    
+    <section class="main">
+        <div class="main-content">
+            <h1>Hey There.I’m Softkoda</h1>
+            <h2>I’am a Software Developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work and experience!
+             If you like what you see and have a project you need coded, 
+             don’t hestiate to contact me.
+            </p>
+            <ul>
+                <li><img src="images/Vector-1.png" alt=""></li>
+                <li><img src="images/Group.png" alt=""></li>
+                <li><img src="images/hand.png" alt=""></li>
+                <li><img src="images/twitter.png" alt=""></li>
+                <li><img src="images/microverse.png" alt=""></li>
+            </ul>
```

Comment:
- [ ] Kindly make sure to put the social media icons inside a lin tag `a` so the structure should look like this
```
<ul>
        <li>
            <a href=""><img src="images/Vector-1.png" alt=""></a>
        </li>
        ...
        
```